pkg/agent/openflow/multicluster.go

```diff
@@ -96,6 +101,19 @@ func (f *featureMulticluster) l3FwdFlowToRemoteViaTun(
 			Action().LoadRegMark(ToTunnelRegMark).
 			Action().GotoTable(L3DecTTLTable.GetID()).
 			Done(),
+		// This generates the flow to forward cross-cluster reject traffic based
+		// on Gateway IP and reg.
+		L3ForwardingTable.ofTable.BuildFlow(priorityNormal-1).
```

Comment:
I suppose you should only add this flow when StretchNetworkPolicy is enabled?

Reply:
Good one. Done.